binary_transparency/firmware/cmd/ft_personality/internal/http/server_test.go

```diff
@@ -79,7 +81,27 @@ func TestRoot(t *testing.T) {
 	}
 }
 
+func b64Decode(b64 string) []byte {
```

Comment:
just a thought - if you add a first parameter of `t *testing.T` to this function, then call `t.Helper()` on the first line of this func, it'll help go show more useful info if it were to fail.

Reply:
Good Suggestion, now Done!